pkg/eventcache/eventcache.go

```diff
@@ -21,9 +21,9 @@ import (
 
```

Comment:
>  Make eventRetryTimer and cacheStrikes from eventcache public 

Maybe add something like:

> 
> We do this so that we can use the value in the json checker and in testing. See next two commit for details.

in the commit message?